Assets/FamilySetupTest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using UnityEngine;
+
+public class FamilySetupTest : MonoBehaviour
+{
+	void Start()
+	{
+		UIApi.OpenWindow(WindowTypes.CharacterSetup); 
+		UIApi.OpenWindow(WindowTypes.FamilySetup);
+	}
+
+	void Update()
+	{
+		Debug.Log("Family Created: " + Properties.family.name);
+		Debug.Log("Number of persons in this family: " + Properties.family.persons.Count);
+	}
```

Comment:
This entire function is for debugging purposes.
Before a branch may be merged, all debugging leftovers need to be removed.